docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "2"
+services:
+  postgres:
+    image: postgres
+    container_name: pg-docker
```

Comment:
It is better to not specify `container_name` in docker-compose.yml to avoid possible conflicts with other containers. Docker will generate new name based on service and image name.